src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOpTPAdd.java

```diff
@@ -45,4 +48,18 @@ public void visit( final LogicalPlanVisitor visitor ) {
 		visitor.visit(this);
 	}
 
+	@Override
+	public String toString(){
+		final StringBuilder builder = new StringBuilder();
+
+		builder.append("> tpAdd ");
+		builder.append("( ");
+		builder.append( tp.asJenaTriple() );
```

Comment:
```suggestion
		builder.append( tp.asJenaTriple().toString() );
```
Can you please use this as a general rule and always call the `toString()` method explicitly.